docs/src/architecture/README.md

```diff
@@ -24,5 +24,5 @@ This is an optional feature, and the components are free to also use cloud speci
 3. The mapper processes are responsible for translating the canonical data format into cloud specific messages and vice versa.
 
 
-![Overview](./thin-edge-overview.png)
+![Overview](./images/thin-edge-overview.png)
```

Comment:
@makr11st  Like this? Then we don't need to add a new image.

```suggestion
You can find the diagrams which explain how those 3 components interact with each other from [Software Management Speficication](https://thin-edge.github.io/thin-edge.io-specs/software-management/sm-agent.html).
```

Reply:
Yes, but I would leave this diagram as it is (general overview) and add the other link in software-management.md

Reply:
Oh no! Here's not what I meant!!! :D:D:D